builders/build/templates.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Templates
+description: Tanssi includes templates to kick-start the development of an Appchain, one for a Substrate-oriented runtime and another featuring full EVM (Ethereum) support.
+---
+
+# Templates {: #templates } 
+
+## Introduction {: #introduction }
+
+ContainerChains deployed through Tanssi are essentially parachains within the Polkadot ecosystem that are capable of interacting with the relay chain and other parachains. They also contain the necessary functionalities to support the Tanssi protocol.
+
+As presented in the [Included Templates](/learn/tanssi/included-templates#baseline-appchain-template){target=_blank} from the Learn section, Tanssi already provides two templates to jumpstart the development process:
+
+- **[Baseline Appchain template](/learn/tanssi/included-templates/#baseline-appchain-template){target=_blank}** - a template that provides the basic platform to start adding custom logic in a Substrate based Appchain
+- **[Baseline EVM (Ethereum Virtual Machine) Template](/learn/tanssi/included-templates/#baseline-evm-template){target=_blank}** - a template that provides full Ethereum compatibility
+
+In this article, the required base setup and how to use the templates as a starting point to start building your Appchain are presented.
+
+## Base Setup - Integrating Into Polkadot's ecosystem {: #base-setup-integrating-into-polkadot }
```

Comment:
I like this!

But it feels kind of weird saying that you need just Cumulus and that is it. Maybe fill this a bit more explaining idk something so that it just does not feel like "Put cumulus, that is it"